tests/Microsoft.Bot.Builder.Azure.Tests/CosmosDbPartitionStorageTests.cs

```diff
@@ -185,6 +188,13 @@ public async Task WritingNoStoreItemsDoesntThrow()
             await _storage.WriteAsync(changes);
         }
 
+        // NOTE: THESE TESTS REQUIRE THAT THE COSMOS DB EMULATOR IS INSTALLED AND STARTED !!!!!!!!!!!!!!!!!
+        [IgnoreOnNoEmulatorFact]
+        public async Task DeletingNullStoreItemsThrowException()
+        {
+            await Assert.ThrowsAsync<ArgumentNullException>(() => _storage.DeleteAsync(null));
+        }
+
         // NOTE: THESE TESTS REQUIRE THAT THE COSMOS DB EMULATOR IS INSTALLED AND STARTED !!!!!!!!!!!!!!!!!
         // For issue https://github.com/Microsoft/botbuilder-dotnet/issues/871
```

Comment:
Duplicated comment line?

Reply:
All tests have it, to keep consistency between them, i just left it.